uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -216,6 +216,7 @@ define([
 
         doc.activity_date_time = $scope.parseDate(doc.activity_date_time) || new Date();
         doc.expire_date = $scope.parseDate(doc.expire_date);
+        doc.status_id = (!$scope.isRole('admin')) ? '2' : $scope.document.status_id; // 2 => 'Awaiting Approval'
```

Comment:
1. `(!$scope.isRole('admin'))` the outer parenthesis are not necessary `!$scope.isRole('admin')`
2. Why is the status id hardcoded?
3. So if i'm staff or manager the id is always going to be "Awaiting approval"?